### ISSUE TITLE
Neon wallet version 2.11 on the MAc desktop suddenly cuts off access to NEO and GAS tokens listed on Dora NEO N3

### ISSUE THREAD

**Issue by lgourlay112** [CityOfZion/neon-wallet#2353]
Please complete the form below. Any issue added that does not include these details will be closed as "not enough information".

For support using the software or concerns about the status of your funds, please go to the #support channel on the NEO Smart Economy Discord Server - https://discord.cityofzion.io

**Wallet version or commit SHA:**
e.g.: 0.2.3 or dcae88c

**Environment:**
e.g.: Win 10 Home (64 bit) Version 1709 Build 16299.309

**Prerequisites:**
e.g.: Don't have any custom tokens added before running the wallet

**Reproduction steps:**
e.g.:
1. Start Neon Wallet
2. Click on "Manage Neon settings" button
3. Click on "Manage Tokens" button
4. Click on "Add a new token" button

some description (for ex. what happens after completing reproduction steps), e.g.:
After clicking on "Add a new token" button, wallet "hangs" for 2 seconds.

**Screenshots/Video:**
e.g.: https://example.com/screenshot.png

**Comment by comountainclimber** [comment]
Could you please provide more details as to what exactly the issue is

**Comment by lgourlay112** [comment]
I upgraded to Neon wallet version 2.11 as prompted in Firefox. When I got
into my wallet and clicked on the migration tab, it took me to the "view
migration history". When I clicked on the amount of Neo that I migrated
tab, it shows "migration details" and then "Processed on Neo N3" , an
address and then "view on dora". I click the "view on dora", and the screen
goes black but today it did not do so for the first time since I upgraded.
I now see the total number of NEO and GAS that I own, under "balance",


after clicking on the final destination address.

So my question now is how can I receive or send more NEO or GAS?
